kaggle_sets/data_preparation/train_test_split.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd
+from pathlib import Path
+import numpy as np
+
+
+"""
+Usage:
+    dtnp = DatasetToNumpy("mosquito-indicator", ',')
+    x, y = dtnp(["date"], "mosquito_Indicator")
+"""
+
+
+class DatasetToNumpy:
+    def __init__(self, csv_file: str, csv_delimeter: str) -> None:
+        self.csv = Path(f"./datasets/{csv_file}.csv")
+        self.csv_delimeter = csv_delimeter
+
+    def __call__(self, drop_list: list, y_column: str, test_size: float = 0.2, random_seed: int = 42) -> tuple:
+        return self._cast(drop_list, y_column, test_size, random_seed)
+        
+    def _cast(self, drop_list: list, y_column: str, test_size: float, random_seed: int) -> tuple:
+        dataframe = pd.read_csv(self.csv, delimiter=self.csv_delimeter)
+
+        drop_list.append(y_column)
+        x = dataframe.drop(drop_list, axis=1)
+        y = dataframe[y_column]
+        
+        np_x = x.to_numpy()
+        np_y = y.to_numpy()
+        np_y = np.expand_dims(y, axis=1)
+
+        np.random.seed(random_seed)
```

Comment:
I suppose would be better to move your code into separate function and call it in ```_cast``` method

Reply:
Done